js/index.js

```diff
@@ -137,3 +138,115 @@ function creationOwlCarousel () {
             e.preventDefault();
     });
 };
+
+/**Coding section for Cards */
+if (document.readyState == 'loading') {
+    document.addEventListener('DOMContentLoaded', ready)
+} else {
+    ready();
+}
+
+function ready() {
+    let removerCartItemButtons = document.getElementsByClassName('btn-danger');
+    //console.log(removerCartItemButtons);
+    for (let i = 0; i < removerCartItemButtons.length; i++){
+        let button = removerCartItemButtons[i];
+        button.addEventListener('click', removeCartItem);
+    }
+
+    let quantityInputs = document.getElementsByClassName('cart-quantity-input');
+    for (let i = 0; i < quantityInputs.length; i++) {
+        let input = quantityInputs[i];
+        input.addEventListener('change', quantityChanged);
+    }
+
+    let addToCartButtons = document.getElementsByClassName('add-to-cart')
+    for (let i = 0; i < addToCartButtons.length; i++) {
+        let button = addToCartButtons[i];
+        button.addEventListener('click', addToCartClicked);
+    }
+
+    document.getElementsByClassName('btn-clear')[0].addEventListener('click', clearClicked)
+}
+
+function clearClicked() {
+    let cartItems = document.getElementsByClassName('cart-items')[0]
+    if (cartItems.firstChild || cartItems.childNodes.length ) {
+    
+    } else {
+        alert('You did not add products yet');
+    }
+
+    while ( cartItems.hasChildNodes()) {
+        cartItems.removeChild(cartItems.firstChild);
+    }
+    updateCartTotal();
+}
+
+function removeCartItem(e) {
+    let buttonClicked = e.target
+    buttonClicked.parentElement.parentElement.remove()
+    updateCartTotal();
+}
+
+function quantityChanged(e) {
+    let input = e.target
+    if (isNaN(input.value) || input.value <= 0) {
+        input.value = 1
+    } 
+    updateCartTotal();
+}
+
+function addToCartClicked(e) {
+    let button = e.target;
+    let productItem = button.parentElement.parentElement.parentElement.parentElement;
```

Comment:
4 times parent? I think we need to rewrite it, because right now you heavily rely on the document structure and it should not be the case.